tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -19,7 +19,8 @@
    "metadata": {},
```

Comment:
<p>I think this is not how we want to edit this. I changed this in the conventions notebook too, perhaps revert it here otherwise we'll have a conflict</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I have seen how it is done there, it will be indeed best to merge conventions and then state preparation</p>